packages/elements/package.json

```diff
@@ -96,5 +96,8 @@
   },
   "release": {
     "extends": "@stoplight/scripts/release"
+  },
+  "exports": {
```

Comment:
Aren't you missing the equivalent for `dev-portal`?

Reply:
Yeah, right. We also export styles in dev-portal. I'll add it